packages/renderer/src/stores/navigation/navigation-registry.ts

```diff
@@ -85,13 +82,6 @@ function collecItem(navigationRegistryEntry: NavigationRegistryEntry, items: Dis
   if (items.find(i => i.name === navigationRegistryEntry.name)) {
     return;
   }
-
-  if (navigationRegistryEntry.type !== 'section') {
```

Comment:
I don't have to remove the content, but keep it, as the condition is `!section`

Reply:
🤦🏼‍♂️ Sorry, good catch. Fixed.